install-manifests/docker-compose-traefix-tls-gzip/README.md

```diff
@@ -0,0 +1,29 @@
+# Hasura GraphQL Engine on Docker with Traefix (Reverse Proxy and Load Balancer)
+
+This Docker Compose setup runs [Hasura GraphQL Engine](https://github.com/hasura/graphql-engine) along with Postgres, Platypus (authentication), and Traefix (a reverse proxy and load balancer, it's officially classified as an edge router) using `docker-compose`.
+
+The platypus and Hasura engine will be both serving with TLS enabled behind _Traefix_. They will have HSTS (TLS Security headers) enabled and gzip compression enabled, and CORS enabled. These are confiurable from the `labels` section in the `docker-compose.yml`.
+
+## Pre-requisites
+
+- [Docker](https://docs.docker.com/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+## Usage
+
+- Clone this repo on a machine where you'd like to deploy graphql engine
+- Edit the environment variables in the `.env` file.
+  - **NOTE**: Your public key and private key have to be formatted with newlines. This can be achieved with `awk -v ORS='\\n' '1' public_key.pem`.
```

Comment:
It will be good to indicate why the keys are required - for JWT authentication.

Reply:
Done.